ampd/src/lib.rs

```diff
@@ -823,6 +945,8 @@ pub enum Error {
     Connection,
     #[error("task execution failed")]
     Task(#[from] TaskError),
+    #[error("App (task group) execution failed")]
```

Comment:
```suggestion
    #[error("app failed")]
```
the error will look like this: `app failed: task group failed: task failed: ...`